nbd/tgt_nbd.cpp

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0
 
 #include <config.h>
+#include <climits>
```

Comment:
I'd suggest to move it into ublksrv_tgt.h, since any target should need it.

Thanks,

Reply:
Cool.  I've changed the `ublksrv_tgt.h` common header to use `<limits.h>`, (not `<climits>`) since it tries to be C-friendly, so it may not be used with a C++ compiler.